api/STYLE.md

```diff
@@ -116,7 +116,7 @@ organization](#package-organization) above.
 To add an extension config to the API, the steps below should be followed:
 
 1. If this is still WiP and subject to breaking changes, please tag it
-   `option (udpa.annotations.file_status).work_in_progress = true;` and
+   `option (xds.annotations.v3.file_status).work_in_progress = true;` and
    optionally hide it from the docs (`[#not-implemented-hide:]`.
```

Comment:
nit: not a change in this PR - but this sentence is missing a closing parenthesis

Reply:
Added it.